gradle.properties

```diff
@@ -2,9 +2,9 @@ mod_version = 1.5.2
 group = net.shadowfacts
 archivesBaseName = Forgelin
 
-mc_version = 1.12
-mcp_mappings = snapshot_20170629
-forge_version = 14.21.1.2387
+mc_version = 1.12.2
```

Comment:
this should be on 1.12 still... to make sure 1.12, 1.12.1 and 1.12.2 all work well with it.

building against 1.12.2 would probably not be that bad... but only if this version doesn't get enforced as the minimum required version.

Reply:
Sure, I can revert this